Makefile

```diff
@@ -38,6 +38,7 @@ else
 		platform := Linux
 		CXX ?= g++
 		linkFlags += -l GL -l m -l pthread -l dl -l rt -l X11
+		libGenDir := src
```

Comment:
Given that all three platforms now generate their static libs to `src`, we can now go ahead and remove all usages of `libGenDir` across the Makefile, if you don't mind making that change here.

Reply:
I was thinking making it a variable that all platforms share, as in putting it on top before individual platform checks, but removing the variable entirely may seem like a cleaner option for now.